src/Command/AllCommand.php

```diff
@@ -16,6 +16,9 @@
  */
 namespace Bake\Command;
 
+use Bake\Command\ControllerCommand;
+use Bake\Command\ModelCommand;
+use Bake\Command\TemplateCommand;
```

Comment:
Just dont use these use statements
They are not needed.

Reply:
If I take them out, the unit tests fail.

Reply:
Ah.. never mind. You're right.

Reply:
Removing them didn't solve the problem though :) same failure.